browser/src/Services/QuickOpen.ts

```diff
@@ -62,10 +67,12 @@ export class QuickOpen {
                         })
                 } else {
                     // TODO: This async call is being dropped, if we happen to use the promise
-                    return recursiveQ(process.cwd())
-                        .then((files: string[]) => {
-                            this._showMenuFromFiles(files)
-                        })
+                    return glob("**/*", {
+                        nodir: true,
+                        ignore: ["**/node_modules/**"], // all hidden dirs (start with '.') are skipped by default
+                    }, (_err: any, files: string[]) => {
+                        this._showMenuFromFiles(files)
```

Comment:
If there is an error, files might be empty, and then QuickOpen crashes for me. Perhaps we should validate we actually have an array for `files` somewhere